core/src/main/java/org/arquillian/smart/testing/configuration/ObjectMapper.java

```diff
@@ -158,6 +160,8 @@ private static Object convert(Method method, Object mappedValue) {
             return mappedValue;
         } else if (ConfigurationSection.class.isAssignableFrom(parameterType)) {
             return mapToObject((Class<ConfigurationSection>) parameterType, (Map<String, Object>) mappedValue);
+        } else if (parameterType.isAssignableFrom(String.class) && List.class.isAssignableFrom(mappedValue.getClass())) {
+            return ((List) mappedValue).stream().collect(Collectors.joining(", "));
```

Comment:
could you shed more light on this case? which use-case are you covering by this?

Reply:
We need to set list of string to string for `inclusion` & `exclusion` from `affected` configuration.

you can find respective test it [here](https://github.com/arquillian/smart-testing/pull/243/files#diff-4b3896eea6384f8873440fb7836cbfa0R86).

```yaml
strategiesConfiguration:
    affected:
      config: affected-configuration.properties 
      transitivity: true 
      exclusions:
         - org.package.*
         - org.arquillian.package.*
      inclusions:
         - org.package.exclude.*
         - org.arquillian.package.exclude.*
```

OR

```yaml
strategiesConfiguration:
    affected:
      config: affected-configuration.properties 
      transitivity: true 
      exclusions: org.package.*,  org.arquillian.package.*
      inclusions: org.package.*, org.arquillian.package.exclude.*
```
Here in yaml configuration file we have user can provide `inclusion` & `exclusion` in comma separated string or list.

Reply:
wouldn't make more sense to keep it just as a list? The string has to be split anyway - to process it (I guess).

Reply:
Now I understand why you did this - you were trying to keep the same way as it was done previously - in the `affected-configuration.properties` but that was properties file, not YAML file.